### ISSUE TITLE
How to use “Log into Oxygen Logfile”？

### ISSUE THREAD

**Issue by anka120** [DEWETRON/OXYGEN-SDK#10]
Since ‘Oxygen 6.0’ has been published.I wonder how to use the fuction of "Log into Oxygen Logfile"?Thanks.

**Comment by matthiasstraka** [comment]
There is a special message for logging in the ODK. You need to send an `odk::host_msg::LOG_MESSAGE` message (either sync or async) with an `odk::IfStringValue` parameter to the host interface. The `key` parameter is a severity level with fatal = 1, error = 2, warning = 3, info = 4.

For example, when you have an `IfHost* host`, you can log an Info (4) text to it using the following code:
```cpp
auto log_msg = host->createValue<odk::IfStringValue>();
log_msg->set("Example log message async");
host->messageAsync(odk::host_msg_async::LOG_MESSAGE, 4, log_msg.get());
```
or
```cpp
auto log_msg = host->createValue<odk::IfStringValue>();
log_msg->set("Example log message sync");
host->messageSync(odk::host_msg::LOG_MESSAGE, 4, log_msg.get());
```
with a sync message call.

We will add this information to the documentation if it is not already there. Also, we will provided an enum/define for the severity level which appears to be missing in the GitHub repository.

**Comment by matthiasstraka** [comment]
I added a new header `odk/api/inc/odkapi_logging.h` to the repository. It is not available in the Release 6.0 version but can be copied there manually if needed. In later releases, it will be included automatically.

This allows to write the severity level as an enum:
```
#include "odkapi_logging.h"
// ...
host->messageAsync(odk::host_msg_async::LOG_MESSAGE, odk::LOGLEVEL_INFO, log_msg.get());
```

Please let me know if your issue can be resolved now.

**Comment by anka120** [comment]
Yes,it works well.Thanks.
Another quesions:
1.Could I add notice like this,when customer edit the config in function of 'update()'?
![capture](https://user-images.githubusercontent.com/42789439/148914397-a7c5cf82-f96d-4529-8472-fd0ce1e74efa.PNG)

**Comment by matthiasstraka** [comment]
Unfortunately, this feature is currently not included in Oxygen 6.0. There has been preliminary work on status messages for a future Version of Oxygen, so you can expect it to work in Oxygen 6.1 or later. I will create a new GitHub Issue so we can track progress on it.

**Comment by anka120** [comment]
Ok，thanks
